src/app/lib/analyticsUtils/index.js

```diff
@@ -14,6 +14,8 @@ export const getDestination = statsDestination => {
     NEWS_GNL_TEST: 598288,
     WS_NEWS_LANGUAGES: 598342,
     WS_NEWS_LANGUAGES_TEST: 598343,
+    DEFAULT: 596068,
```

Comment:
Are `DEFAULT` and `DEFAULT_TEST` supposed to have the same bucket numbers?

Reply:
Yes - there isn't a separate Test for Default, as it's the 'everything else' bucket.

Reply:
Doesn't that mean that visits from `https://www.test.bbc.com/scotland` would be in the same bucket as `https://www.bbc.com/scotland` and cloud the user metrics, since they'd have the same page metadata?

Reply:
Yes, but I expect the change where /scotland data goes before they actually start using it - am just waiting for someone to decide what the correct one will be, but made this change to stop the data ending up in News.

Reply:
@jimjohnsonrollings Can you please open an issue to update the destination for `/scotland` and remove the `default` bucket and link it here?

Reply:
https://github.com/bbc/simorgh/issues/3810